search/ranking_info.cpp

```diff
@@ -77,10 +72,19 @@ double constexpr kPoiType[] = {
   0.01,       // Hotel
   0.01,       // Shop or Amenity
   0.01,       // Attraction
+  0.005,      // PureCategory
   0,          // General
  -0.01,       // Service
 };
-static_assert(std::size(kPoiType) == base::Underlying(PoiType::Count));
+static_assert(std::size(kPoiType) == base::E2I(PoiType::Count));
+
+// - When search for "eat", we'd prefer category types, rather than "eat" name.
+// - To _equalize_ "subway" search: Metro category should be equal with "Subway" fast food.
```

Comment:
Isn't it better/100% reliable to make a list of translations/terms together with "is category" boolean to filter out 100% of all non-relevant matches?

Reply:
This is like an obvious and famous example. There are no non-relevant matches here.
Searching for "subway" should match food and metro equally (except distance).

Reply:
When someone types Subway in the US, they expect to see eating places in the first place. Searching for all subway stops is not well known or popular query at all.

Reply:
"Subway" fast food has a small priority over the subway - 1km if converted to distance.